core/src/test/java/greencity/mapping/AddEcoNewsDtoRequestMapperTest.java

```diff
@@ -3,67 +3,29 @@
 import greencity.ModelUtils;
 import greencity.dto.econews.AddEcoNewsDtoRequest;
 import greencity.entity.EcoNews;
-import greencity.exception.exceptions.LanguageNotFoundException;
-import greencity.repository.UserRepo;
-import greencity.service.LanguageService;
-import greencity.service.TagService;
 import java.util.Collections;
-import java.util.Optional;
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
-import static org.mockito.ArgumentMatchers.anyString;
 import org.mockito.InjectMocks;
-import org.mockito.Mock;
-import static org.mockito.Mockito.when;
 import org.mockito.junit.MockitoJUnitRunner;
 
 @RunWith(MockitoJUnitRunner.class)
```

Comment:
There is already Junit5. When writing new tests try to use Junit5 and refactor old test when you have time and making new commits